gulpfile.js

```diff
@@ -67,13 +66,30 @@ gulp.task('copy-content-assets', () => {
       // they belong to.
       .pipe(
         rename((assetPath) => {
-          const leftPart = assetPath.dirname.split('/')[1];
-          // Let certain path's images to pass through.
+          const parts = assetPath.dirname.split('/');
+          // Landing pages should keep their assets.
+          // e.g. en/vitals, en/about
+          if (parts.length === 2) {
+            return;
+          }
+
+          // Let images pass through if they're special, i.e. part of the
+          // handbook or newsletter. We don't treat these URLs like the
+          // rest of the site and they're allowed to nest.
+          const subdir = parts[1];
           const imagePassThroughs = ['handbook', 'newsletter'];
-          if (imagePassThroughs.includes(leftPart)) {
+          if (imagePassThroughs.includes(subdir)) {
             return;
           }
-          return (assetPath.dirname = path.basename(assetPath.dirname));
+
+          // Some assets are nested under directories which aren't part of
```

Comment:
This feel fragile, but I suppose the entire way we do permanent URLs for posts is (lots of hardcoded 'handbook' and 'newsletter'). Just my 2c.

Reply:
Actually, it's not hardcoded that much, but I still think it might not be immediately obvious.

Reply:
100% agree it's not good. In fact the fragility is evident because this used to work but then we broke it without really noticing.

I think we should flatten the directory structure so the output matches the src. Ewa's also working towards this for i18n reasons. However, I'm concerned about making that big of a change during the run up to web.dev/live because it'll mean needing to rebase all inflight content PRs and update what folders they target.

I'll put this on the agenda for our next eng sync for us to discuss. In the meantime I'd like to land this PR to at least fix what's currently broken.